server/config/config.go

```diff
@@ -54,13 +54,13 @@ const (
 	// GrpcServiceKeepAlivePingMinIntervalSec controls the min interval for one keepalive ping.
 	defaultGrpcServiceKeepAlivePingMinIntervalSec int = 20
 
-	defaultNodeNamePrefix          = "ceresmeta"
+	defaultNodeNamePrefix          = "horaemeta"
 	defaultEndpoint                = "127.0.0.1"
 	defaultRootPath                = "/ceresdb"
 	defaultClientUrls              = "http://0.0.0.0:2379"
 	defaultPeerUrls                = "http://0.0.0.0:2380"
 	defaultInitialClusterState     = embed.ClusterStateFlagNew
-	defaultInitialClusterToken     = "ceresmeta-cluster" //#nosec G101
+	defaultInitialClusterToken     = "horaemeta-cluster" //#nosec G101
```

Comment:
There may be a compatibility issue here.

Reply:
Only the default cluster name has been changed, theoretically there should be no compatibility issues.